src/libs/SidebarUtils.ts

```diff
@@ -311,6 +311,7 @@ function updateReportsToDisplayInLHN({
     for (const reportID of updatedReportsKeys) {
         const report = reports?.[reportID];
         if (!report?.reportID && report?.pendingFields?.reportID !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE) {
+            delete displayedReportsCopy[reportID];
```

Comment:
@jnowakow Are you sure there were no other cases where we might want to still show the report in the LHN when this conditions runs? It sounds odd we would not be deleting the reports already